microservices/s3_to_redshift/ddl/servicebc.call_centre_esb_data.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE IF NOT EXISTS servicebc.call_centre_esb_data (
+  "ID" DECIMAL(13,0) NOT NULL PRIMARY KEY ENCODE ZSTD,
+  "SkillGroup" VARCHAR(50) ENCODE ZSTD,
+  "ContactReason" VARCHAR(50) ENCODE ZSTD,
+  "ContactReasonDetail" VARCHAR(50) ENCODE ZSTD,
+  "DateKey" INTEGER ENCODE ZSTD,
+  "INTEGERerval" VARCHAR(5) ENCODE ZSTD,
```

Comment:
@BJenning this looks like what you're referring to. My mistake due to a Find and Replace. I'll check that the table isn't actually this, and fix it if it is. I'll fix the code now.

Reply:
The table is indeed also incorrect. Fixing.